odoo/addons/test_lint/tests/lint_case.py

```diff
@@ -16,7 +16,23 @@ def iter_module_files(self, *globs):
         """
         for modroot in map(get_module_path, get_modules()):
             for root, _, fnames in os.walk(modroot):
-                fnames = [j(root, n) for n in fnames]
+                fnames = [os.path.join(root, n) for n in fnames]
```

Comment:
Why do that? The only results are triggering a security altert and making the loop slower.

Reply:
I just wanted to remove the E402 warning from this file, and since the performances are not critical at all here, I favored readability.

But ok, I'm reverting...